shell/app/node_main.cc

```diff
@@ -113,6 +113,10 @@ int NodeMain(int argc, char* argv[]) {
     gin_helper::Dictionary versions;
     if (process.Get("versions", &versions)) {
       versions.SetReadOnly(ELECTRON_PROJECT_NAME, ELECTRON_VERSION_STRING);
+
+      // We don't build with OpenSSL
+      if (versions.Has("openssl"))
+        versions.Delete("openssl");
```

Comment:
Do we need `Has()`? I think `Delete()` should just no-ops if the key doesn't exist.

Reply:
oh i think you're right - will update.